tests/packages/msi.template/templates/ReadMe.md

```diff
@@ -46,7 +46,7 @@ functions that you can use, these are sometimes called the helpers.
 https://chocolatey.org/docs/helpers-reference
 
 A note about a couple:
-* Get-BinRoot - this is a horribly named function that doesn't do what new folks think it does. It gets you the 'tools' root, which by default is set to 'c:\tools', not the chocolateyInstall bin folder - see https://chocolatey.org/docs/helpers-get-tools-location
+

```

Comment:
Is this intended to be an empty line? (It seems it should, but it wasn't there initially...)